pkg/api/routes.go

```diff
@@ -54,10 +54,10 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
-			"GetImageMapInfo",
+			"GetClientAccessInfo",
 			"GET",
-			"/image/mapinfo",
-			h.GetImageMapInfo,
+			"/client",
```

Comment:
i'm not sure how exactly RGW creds are going to be surfaced yet, but this makes sense.  I will do that refactor when I'm implementing `rook object` commands.